include/openenclave/bits/result.h

```diff
@@ -372,6 +372,23 @@ typedef enum _oe_result
      */
     OE_QUOTE_LIBRARY_LOAD_ERROR,
 
+    /**
+     * Failed to load the SGX quote-ex library used for quote generation and
+     * attestation.
+     */
+    OE_SGX_QUOTE_EX_LIBRARY_LOAD_ERROR,
+
+    /**
+     * The SGX AESM service used for quote generation and attestation is not
+     * available.
+     */
+    OE_SGX_AESM_UNAVAILABLE,
+
+    /**
+     * The SGX quote-ex library API failed.
+     */
+    OE_SGX_QUOTE_EX_LIBRARY_ERROR,
```

Comment:
Do we expect that every TEE will add its own error codes?

Reply:
Good point. Will combine them with generic error codes and raise more clear error messages.